### PR TITLE
Fixed use of keyidentifier

### LINKED ISSUE #6: keystrokes (key/down events) don't seem to be recorded
https://www.youtube.com/watch?v=AazGdGUw4kw

### PR DESCRIPTION
**Pull request by mrawdon**
This fixes #6
keyIdentifier was replaced in favor of key https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyIdentifier

### CODE REVIEW

File: lib/EventProxy.js [hunk 1]
```diff
@@ -172,7 +172,7 @@
 			if (event.type === 'click' && isDragEvent()) {
 				return;
 			}
-
+
```
Comment (dylans) [inline lib/EventProxy.js]: extra whitespace?

**Review by dylans — CHANGES_REQUESTED** [review]
minor whitespace nit, otherwise looks fine to me.

**Comment by jason0x43** [comment]
This will break Safari, which still uses `keyIdentifier`. We'll need to try both `keyIdentifier` and `key` when getting the value from the event.

**Comment by mrawdon** [comment]
I added a fallback for keyIdentifier if key isn't defined

**Review by dylans — COMMENTED** [review]
Are all other instances of event.key working through the EventProxy? (I've not had time to verify...) If so, this looks fine, otherwise there are a few other places that need similar treatment.

**Comment by mrawdon** [comment]
As far as I could tell this was the only place where the key was pulled from the event in EventProxy.  It works for me in Chrome I'm not able to test Safari though

**Review by The-Alchemist — APPROVED** [review]

**Review by dylans — APPROVED** [review]

**Comment by wizzard0** [comment]
and... was it eventually merged?

**Comment by jason0x43** [comment]
Merged in a9f7c14f68a18752ca1b07dea530f0835dff061b. Thanks!

**Pull request closed without merging**